spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -176,6 +176,38 @@
         .to redirect_to(Settings.saml.logout_relay + '?success=true')
     end
 
+    context 'SSO cookie' do
```

Comment:
NITPICK but it probably wouldn't hurt to test the configuration whereby setting cookies is disabled.

Also, this just occurred to me, but lets say we have cookie enabled and then we toggle to false, should we always destroy this cookie when it is encountered in that scenario, probably not a big deal seeing as it is short lived anyway. Just a thought that occurred to me thought I would mention.